cypress/integration/article_spec.js

```diff
@@ -14,10 +16,8 @@ describe('News Article', () => {
     })
 })
 
-describe('Renderer Status', () => {
+describe('Page Status', () => {
     it('should display 200/OK', () => {
-        cy.request('/status').then((response) => {
-            expect(response.status).to.eq(200)
-        })
+      testRequestResponse('/', 200)  
     })
 })
```

Comment:
Probably worth adding the renderer status as well as the page status. However it could be handled in a seperate ticket

Reply:
Rather than handle the /status endpoint in a separate ticket, i think just move it to a different spec file.

Reply:
I agree with Jordan